demo/esmodule.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Advanced ES Module Demo</title>
+
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+  <link rel="stylesheet" href="demo.css" />
+
+  <script type="module" src="https://unpkg.com/ionicons@4.5.10-0/dist/ionicons/ionicons.esm.js"></script>
+  <script nomodule="" src="https://unpkg.com/ionicons@4.5.10-0/dist/ionicons/ionicons.js"></script>
+
+  <style type="text/css">
+    .grid-stack-item-removing {
+      opacity: 0.8;
+      filter: blur(5px);
+    }
+    #trash {
+      background: rgba(255, 0, 0, 0.4);
+    }
+  </style>
+</head>
+
+<body>
+  <h1>Advanced ES Module Demo</h1>
+  <div class="row">
+    <div class="col-md-2 d-none d-md-block">
+      <div id="trash" style="padding: 5px; margin-bottom: 15px;" class="text-center">
+        <div>
+          <ion-icon name="trash" style="font-size: 300%"></ion-icon>
+        </div>
+        <div>
+          <span>Drop here to remove!</span>
+        </div>
+      </div>
+      <div class="newWidget grid-stack-item">
+        <div class="grid-stack-item-content" style="padding: 5px;">
+          <div>
+            <ion-icon name="add-circle" style="font-size: 300%"></ion-icon>
+          </div>
+          <div>
+            <span>Drag me in the dashboard!</span>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="col-sm-12 col-md-10">
+      <div class="grid-stack"></div>
+    </div>
+  </div>
+
+  <script type="module">
+    import { GridStack } from '../dist/esm/gridstack.js';
```

Comment:
how can you load that ? in Chrome I get:

Access to script at 'file:///C:/depot/gridstack.js/dist/gridstack.js' from origin 'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, isolated-app, chrome-extension, chrome, https, chrome-untrusted.

Reply:
I got that too; ended up serving the directory from the root of the repo (i.e. `npx http-server .` in `/gridstack.js`)